src/website/src/app/documentation/demos/checkboxes/ng/disabled.html

```diff
@@ -1,4 +1,4 @@
-<clr-checkbox-container>
+`<clr-checkbox-container>
```

Comment:
Do you know what dependency changed this formatting? Prettier remained the same.

Reply:
I updated it when I was getting website build issues. 
Now, I don't seem to be able to revert them back to master, prettier seems to change the files to something different from master. 

Do you think I should just close this PR and submit another one sans formatting changes?